project.clj

```diff
@@ -50,6 +51,14 @@
                               "phantom-advanced" ["phantomjs"
                                                   "src/test/js/runner.js"
                                                   "src/test/html/advanced.html"]}}
+  
+  ;; Factoring out the duplication of this test selector function causes an error,
```

Comment:
This seems like something that isn't really part of Leiningen's expected API.  Do you know of any docs etc. suggesting this approach is expected?

Copying 2 fairly simple lines of code isn't a big deal.  I'd prefer to just avoid any possible difficulties.